src/core/types/addons.js

```diff
@@ -54,6 +55,24 @@ type EventDataType = {|
   conversion: string,
 |};
 
+export type ExternalPreviewType = {|
+  caption: LocalizedString | null,
+  image_size: [number, number],
+  image_url: string,
+  thumbnail_size: [number, number],
+  thumbnail_url: string,
+|};
+
+export type PreviewType = {|
+  h: number,
```

Comment:
If this is our internal type, can we have a better prop name than a single char?

Reply:
Yes, but before I change it I want to confirm whether you are okay with keeping these changes in this patch, or if we should leave them to a follow-up patch.